src/Tools/dotnet-monitor/DiagnosticsMonitorCommandHandler.cs

```diff
@@ -236,6 +237,8 @@ public static IHostBuilder CreateHostBuilder(IConsole console, string[] urls, st
                     services.Configure<DiagnosticPortOptions>(context.Configuration.GetSection(ConfigurationKeys.DiagnosticPort));
                     services.AddSingleton<IValidateOptions<DiagnosticPortOptions>, DiagnosticPortValidateOptions>();
 
+                    services.ConfigureGlobalCounter(context.Configuration);
```

Comment:
Why isn't this just `services.Configure<GlobalCounterOptions>(context.Configuration.GetSection(ConfigurationKeys.GlobalCounter)`? Especially if you are going to make the IntervalSeconds property nullable, you are going to have to use GetValueOrDefault anyway when accessing it.

Reply:
Nevermind, I miss read it.